dev-python/nodeenv/nodeenv-1.3.5.ebuild

```diff
@@ -16,4 +16,5 @@ LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
+# requires network access
 RESTRICT="test"
```

Comment:
POP QUIZ: In future, what will we be able to put here instead?

Reply:
off the top of my head, network-test?

Reply:
Yes, [slated](https://bugs.gentoo.org/show_bug.cgi?id=553696) for EAPI 8!